src/ui/components/Collections.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import Select from 'react-select';
+import axios from 'axios';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow
+} from '@material-ui/core';
+import ResourceCell from './ResourceCell';
+
+const Collections = () => {
+  const [data, setData] = useState({ collectionName: '', headers: [], data: [] });
+  const collections = [
```

Comment:
Is there a way we can expose this on the backend. That way instead of updating multiple places simply modifying `src/storage/collections.js` would update everything

Reply:
yea I was thinking about this but we need to manually add in headers for each collection unless you can think of some generic headers for all the collections.

Reply:
Hmm, yea without refactoring the database collections to be something like {fullUrl: 'blah', resource: {}} it would be hard... I honestly think we should do that but probably not here